extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -200,15 +182,25 @@ describes.realWin(
     });
 
     it('does use inline config when remote src is invalid', async () => {
-      env.sandbox.stub(Services, 'xhrFor').returns({
-        fetchJson() {
-          throw new Error();
-        },
-      });
+      const exampleURL = 'invalidRemoteURL';
+      pageElement.setAttribute('src', exampleURL);
```

Comment:
Is this necessary? I see that `createAmpStoryShoppingConfig()` already sets a 'src' attribute, although on the shopping attachment element instead. Does this attribute actually need to be set for both the page and  attachment elements?

Reply:
Good point, it's not necessary.